app/js/reducers/draftLabOrderReducer.js

```diff
@@ -1,36 +1,78 @@
 import {
-  ADD_DRAFT_LAB_ORDER,
-  DELETE_DRAFT_LAB_ORDER,
-  DELETE_ALL_DRAFT_LAB_ORDERS,
   TOGGLE_DRAFT_LAB_ORDER_URGENCY,
+  ADD_PANEL_TO_DRAFT_LAB_ORDER,
+  ADD_TEST_TO_DRAFT_LAB_ORDER,
+  DELETE_TEST_FROM_DRAFT_LAB_ORDER,
+  DELETE_PANEL_FROM_DRAFT_LAB_ORDER,
 } from '../actions/actionTypes';
 import initialState from './initialState';
 
+/* *Utility -> TODO: Move to utility folder */
+const isValidList = list => (Array.isArray(list) && !!(list.length));
+
+/* *Utility -> TODO: Move to utility folder */
+const filterThrough = (from, to) => from.reduce((itemFrom, item) => {
+  const check = to.map(i => i.id).includes(item.id);
+  const filter = check ? itemFrom.filter(i => i.id !== item.id) : itemFrom;
+  return filter;
+}, from);
+
```

Comment:
I believe it would be nice if you add some comments on what the `filterThrough` utility does.

Reply:
done!